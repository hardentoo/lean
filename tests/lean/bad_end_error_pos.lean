#exit
example (a b c : nat) (f : nat → nat) : f (a + b + c) = f (b + c + a) :=
by ac_refl

example (a b c : nat) (f : nat → nat) : f (a + b + (c * b * a)) = f (b + (a * c * b) + a) :=
by ac_refl

end

example (a b c : nat) (f : nat → nat → nat) : f (b * c) (c * b * a) = f (c * b) (a * c * b) :=
by ac_refl

example (a b c : nat) (f : nat → nat) : f (a + (b * c) + (c * b * a)) = f ((c * b) + (a * c * b) + a) :=
by ac_refl
