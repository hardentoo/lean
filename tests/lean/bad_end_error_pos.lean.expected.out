bad_end_error_pos.lean:1:0: warning: using 'exit' to interrupt Lean
